### PR TITLE
Adding Checkpoint interop VPN guide

### PR DESCRIPTION
**Pull request by ashishverm**
Still to do:

- Couple of images needs to be updated and added.
Awaiting new images from Checkpoint.

Please start the review process.

- [x] Tech writing / styleguide review - @deb-saunders  
- [x] Formatting review for publishing - @jmdobry

### CODE REVIEW

**Comment by ashishverm** [comment]
I signed it!

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
+---
+title: How to Set Up VPN between Checkpoint and Cloud VPN
+description: Learn how to build site-to-site IPSEC VPN between Checkpoint and Cloud VPN.
+author: ashishverm
+tags: Compute Engine, Cloud VPN, Checkpoint firewall
+date_published: 2018-01-03
+---
+
+# Introduction
+
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add Before you begin section here.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+|Remote network IP ranges| The on-premise CIDR blocks connecting to GCP from the VPN gateway.|
+|Local subnetworks|The GCP CIDR blocks connecting on-premise with the VPN gateway.|
+|Local IP ranges| The GCP IP ranges matching the selected subnet.|
+
+To create a route:
+
+1. From the GCP Console, go to **Routes** > **Create a route**.
+1. Enter the parameters. Click **Create**.
+
+![alt_text](Image_3.PNG)
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Global: Same comment as above for values in graphics.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+
+**Note:** Add ingress firewall rules to allow inbound network traffic as per your security
+policy.
+
+# Check Point
+
+To create an Interoperable Device for Google Cloud on the Check Point
+SmartConsole:
+
+**Step 1**. Open SmartConsole > **New** > **More** > **Network Object** > **More** > **Interoperable Device**.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Fix steps

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+Go to **Encryption** and change the Phase 1 and Phase 2 properties according what is specified in the Cipher configuration settings on page 3.
+
+**Note:** Make sure you select Perfect Forward Secrecy (Phase 2). This example refers to IKEv1. You can also use IKEv2 in this scenario.
+
+![alt_text](Image_6.PNG)
+
+**Step 6**. Go to the **Advanced** tab and modify the Renegotiation Time.
+
+**IKE for Phase 1**: 610 minutes
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: add bullets for clarity

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+**IKE for Phase 2**: 10,800 seconds
+
+![alt_text](Image_7.PNG)
+
+**Step 7**. Configure the Access Control Rule Base and Install policy.
+
+For more information, see the R80.10 Site To Site VPN Administration Guide.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add link to guide

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+1. Go to **Topology**, in the VPN Domain section. Select Manually defined.
+1. Click the right to select the desired object.
+1. Click **New** > **Group** > **Simple Group**.
+1. Enter an **Object Name**, click **OK**. Do NOT assign any objects to this group.
+
+![alt_text](Image_13.PNG)
+
+**Step 4**. In `clish`, create a VPN Tunnel Interface (VTI).
+
+![alt_text](Image_14.PNG)
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Missing graphic

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+**Note:** This is not relevant for a Policy Based scenario.
+Add these directional match rules in the VPN column for every firewall rule related to VPN traffic:
+
+    Internal_clear > Google Cloud VPN community name (VPN_Community)
+    Google Cloud VPN community name > Google Cloud VPN community name
+    (VPN_Community) Google Cloud VPN community name (VPN_Community) > Internal_clear
+
+**Step 11**. Install policy.
+
+For more information, see the R80.10 Site To Site VPN Administration Guide.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add link to guide

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 8]
```diff
@@ -0,0 +1,367 @@
+---
+title: How to Set Up VPN between Checkpoint and Cloud VPN
+description: Learn how to build site-to-site IPSEC VPN between Checkpoint and Cloud VPN.
+author: ashishverm
+tags: Compute Engine, Cloud VPN, Checkpoint firewall
+date_published: 2018-01-03
+---
+
+# Introduction
```
[the remark this answers is not in the document]
Comment (ashishverm) [reply tutorials/using-cloud-vpn-with-checkpoint/index.md]: Done

File: tutorials/using-cloud-vpn-with-checkpoint/index.md (hunk 1 above)
Comment (ashishverm) [reply tutorials/using-cloud-vpn-with-checkpoint/index.md]: Done

**Comment by ashishverm** [comment]
thanks for the review, I have updated the doc.
PTAL.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 9]
```diff
@@ -0,0 +1,383 @@
...
+|Description|           |Description of the cloud router.|
+|Network|`to-cp`|The GCP network the cloud router attaches to. Note: This is the network which manages route information.|
+|Region|`europe-west1`|The home region of the cloud router.Note: Make sure the cloud router is in the same region as the sub-networks it is connecting to.|
+|Google ASN|`65000`|The Autonomous System Number assigned to the cloud router. Use any unused private ASN (64512 - 65534, 4200000000 – 4294967294).|
+
+### VPN Tunnel
+
+In Google Cloud Platform Console, select **Networking** > **Interconnect** > **VPN** to create a VPN connection.
+
+![alt_text](Image_9.PNG)
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Move graphic to below table or omit.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 10]
```diff
@@ -0,0 +1,383 @@
...
+**Creating a VPN connection**
+
+|Parameter|Value|Description|
+|---------|-----|-----------|
+|Name|`gcp-to-cp-vpn`|Name of the VPN gateway|
+|Description|`VPN tunnel connection between GCP and Check Point Security Gateway`|Description of the VPN connection|
+|Network|`to-cp`|The GCP network the VPN gateway attaches to **Note**: This network will get VPN connectivity|
+|Region|`europe-west1`|The home region of the VPN gateway **Note**: Make sure the VPN gateway is in the same region as the subnetworks it is connecting to.|
+|IP address|`cloud-ip(35.195.227.26`|The static public IP address used by the VPN gateway. An existing, unused, static public IP address within the project can be assigned, or a new one created.|
+
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add Step 3: Enter the parameters as shown in the following table to create a VPN tunnel:

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 11]
```diff
@@ -0,0 +1,383 @@
...
+![alt_text](Image_10.PNG)
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Move graphic below table or omit.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 12]
```diff
@@ -0,0 +1,383 @@
...
+
+|Parameter|Value|Description|
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Move to below step 2.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 13]
```diff
@@ -0,0 +1,383 @@
...
+|---------|------|-----------|
+|Remote peer IP address|`199.203.248.181`|Public IP address of the on-premise VPN appliance used to connect to Cloud VPN.
+|IKE version|`IKEv2`|The IKE protocol version. You can select IKEv1 or IKEv2.|
+|Shared secret|`secret`|A shared secret for authentication by the VPN gateways. Configure the on-premise VPN gateway tunnel entry with the same shared secret.|
+|Routing options|`Dynamic routing`|Cloud VPN supports multiple routing options for the exchange of route information between the VPN gateways. In this example, Cloud Router and BGP are configured.|
+|BGP session| |BGP sessions enable your cloud network and on-premise networks to dynamically exchange routes|
+
+![alt_text](Image_11.PNG)
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: move to below table or omit

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 14]
```diff
@@ -0,0 +1,383 @@
...
+
+### BGP Sessions
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add Step 4. Enter the BGP session parameters as shown in the following table:
 
Remove BGP Sessions heading

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 15]
```diff
@@ -0,0 +1,383 @@
...
+
+|Parameter|Value|Description|
+|---------|-----|-----------|
+|Name|`gcp-to-cp-bgp`|Name of the BGP session.|
+|Peer ASN|`65002`|Unique BGP ASN of the on-premise router.|
+|Google BGP IP address|`169.254.0.1`|
+|Peer BGP IP address|`169.254.0.2`|
+
+Click **Save and Continue** to complete.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add Step 5.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 16]
```diff
@@ -0,0 +1,383 @@
...
+**Note:** – Add ingress firewall rules to allow inbound network traffic as per your security policy.
+
+## Configuring Checkpoint Firewall
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Check Point

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 17]
```diff
@@ -0,0 +1,383 @@
...
+
+**Create an interoperable device for Cloud VPN on the Check Point SmartConsole**
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Remove bold.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 18]
```diff
@@ -0,0 +1,383 @@
...
+    set bgp external remote-as 65000 peer 169.254.0.1 as-override on
+    set bgp external remote-as 65000 peer 169.254.0.1 holdtime 30
+    set bgp external remote-as 65000 peer 169.254.0.1 keepalive 10
+    set inbound-route- filter bgp-policy 512 based-on- as as 65000 on
+    set inbound-route- filter bgp-policy 512 accept-all- ipv4
+    set route-redistribution to bgp-as 65000 from interface eth1 on
+
+**Step 10**. Configure Directional Rules.
+
+Adding Directional Rules for a Route-Based Scenario
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Either omit "Add Directional ..." sentence and add to Step 10 (Configure Directional Rules for a Route Based Scenario" 
or
Add to note below.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 19]
```diff
@@ -0,0 +1,383 @@
...
+
+1. Open SmartConsole > **Global Properties** > **VPN** > **Advanced**.
+2. Select **Enable VPN Directional Match in VPN Column**.
+
+![alt_text](Image_20.PNG)
+
+**Note:** This is not relevant for a Policy Based scenario.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: If omitting 1st sentence ("Adding Directional Rules ... "), then add to note here:
Directional Rules are relevant to route-based scenarios, not policy-based scenarios.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 20]
```diff
@@ -0,0 +1,383 @@
...
+Add these directional match rules in the VPN column for every firewall rule related to VPN traffic:
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Under Step 10.,
Add Step 3. Add these directional match rules in the VPN column for every firewall rule ...

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 21]
```diff
@@ -0,0 +1,383 @@
...
+|---------|-----------|-----|
+|Name|`gcp-to-cp-vpn`|Name of the VPN gateway|`gcp-to-cp-vpn`|
+|Description|`VPN tunnel connection between GCP and Checkpoint Security Gateway`|Description of the VPN connection|
+|Network|`to-cp`| The GCP network the VPN gateway attaches to. Note: This network will get VPN connectivity|
+|Region|`europe-west1`|The home region of the VPN gateway Note: Make sure the VPN gateway is in the same region as the subnetworks it is connecting to.|
+|IP address|`cloud-ip(35.195.227.26)`|The VPN gateway uses the static public IP address. An existing, unused, static public IP address within the project can be assigned, or a new one created.|
+|Remote peer IP address| `199.203.248.181`|Public IP address of the on-premise VPN appliance used to connect to the Cloud VPN.|
+|IKE version|`IKEv1`|The IKE protocol version. You can select IKEv1 or IKEv2.|
+|Shared secret|`secret`|A shared secret used for authentication by the VPN gateways. Configure the on-premise VPN gateway tunnel entry with the same shared secret.|
+|Routing options|`Static`|Multiple routing options for the exchange of route information between the VPN gateways. This example uses static routing.|
```
[the remark this answers is not in the document]
Comment (ashishverm) [reply tutorials/using-cloud-vpn-with-checkpoint/index.md]: looks like this got changed today.

**Comment by ashishverm** [comment]
thanks for another round of review, incorporated all the comments.
PTAL.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 22]
```diff
@@ -0,0 +1,383 @@
...
+
+## Configuring Cloud VPN
+
+### Configuring Cloud Router
+
+The Google Cloud Router dynamically exchange routes between your Virtual Private Cloud and on-premise networks with Border Gateway Protocol (BGP). For the initial release, Cloud Router supports BGP for Cloud VPN only. Cloud Router works with both legacy networks and sub-networks.
+
+Create a new Cloud router to configure GCP for Site to Site VPN connectivity.
+
+In Google Cloud Platform Console, select **Networking** > **[Cloud Routers](https://console.cloud.google.com/interconnect/routers)** > **Create Router**.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Step 1.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 23]
```diff
@@ -0,0 +1,383 @@
...
+
+Enter the parameters as shown in the following table and click **Create**.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Step 2.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 24]
```diff
@@ -0,0 +1,383 @@
...
+|Parameter|Value|Description|
+|---------|-----------|-----|
+|Name|`gcp-to-cp-router`|Name of the cloud router.|
+|Description|           |Description of the cloud router.|
+|Network|`to-cp`|The GCP network the cloud router attaches to. Note: This is the network which manages route information.|
+|Region|`europe-west1`|The home region of the cloud router.Note: Make sure the cloud router is in the same region as the sub-networks it is connecting to.|
+|Google ASN|`65000`|The Autonomous System Number assigned to the cloud router. Use any unused private ASN (64512 - 65534, 4200000000 – 4294967294).|
+
+![alt_text](Image_8.PNG)
+
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add: 
### Creating a VPN connection

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 25]
```diff
@@ -0,0 +1,383 @@
...
+In Google Cloud Platform Console, select **Networking** > **Interconnect** > **[VPN](https://console.cloud.google.com/interconnect/vpn)** >> **CREATE VPN CONNECTION**.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Add: Step 1.
 
Remove extra > here: ** >> **CREATE VPN CONNECTION**.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 26]
```diff
@@ -0,0 +1,383 @@
...
+
+**Google Compute Engine VPN gateway**
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Remove: **Google Compute Engine VPN gateway**
Add Step 2: Enter the parameters as shown in the following table for the Google Compute Engine VPN gateway:

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 27]
```diff
@@ -0,0 +1,383 @@
...
+|---------|-----|-----------|
+|Name|`gcp-to-cp-vpn`|Name of the VPN gateway|
+|Description|`VPN tunnel connection between GCP and Check Point Security Gateway`|Description of the VPN gateway|
+|Network|`to-cp`|The GCP network the VPN gateway attaches to **Note**: This network will get VPN connectivity|
+|Region|`europe-west1`|The home region of the VPN gateway **Note**: Make sure the VPN gateway is in the same region as the subnetworks it is connecting to.|
+|IP address|`cloud-ip(35.195.227.26)`|The static public IP address used by the VPN gateway. An existing, unused, static public IP address within the project can be assigned, or a new one created.|
+
+![alt_text](Image_9.PNG)
+
+**Tunnels**
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Remove Tunnels.
Add Step 3:  Enter the parameters as shown in the following table for the tunnel:

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 28]
```diff
@@ -0,0 +1,383 @@
...
+|Remote peer IP address|`199.203.248.181`|Public IP address of the on-premise VPN appliance used to connect to Cloud VPN.
+|IKE version|`IKEv2`|The IKE protocol version. You can select IKEv1 or IKEv2.|
+|Shared secret|`secret`|A shared secret for authentication by the VPN gateways. Configure the on-premise VPN gateway tunnel entry with the same shared secret.|
+|Routing options|`Dynamic(BGP)`|Cloud VPN supports multiple routing options for the exchange of route information between the VPN gateways. In this example, Cloud Router and BGP are configured.|
+|Cloud Router|`gcp-to-cp-router`|Select the Cloud router created previously.|
+|BGP session| |BGP sessions enable your cloud network and on-premise networks to dynamically exchange routes|
+
+![alt_text](Image_10.PNG)
+
+### BGP Sessions
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Remove BGP Sessions.
Add Step 4: Enter the parameters as shown in the following table for the BGP sessions:

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 29]
```diff
@@ -0,0 +1,383 @@
...
+
+|Parameter|Value|Description|
+|---------|-----|-----------|
+|Name|`gcp-to-cp-bgp`|Name of the BGP session.|
+|Peer ASN|`65002`|Unique BGP ASN of the on-premise router.|
+|Google BGP IP address|`169.254.0.1`|
+|Peer BGP IP address|`169.254.0.2`|
+
+Click **Save and Continue** to complete.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Step 4.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 30]
```diff
@@ -0,0 +1,383 @@
...
+
+|Name | Value|
+-----|------
+|Check Point Security Gateway(external IP)|`199.203.248.181`|
+|Addresses behind Check Point Security Gateway|`10.0.0.10/24`|
+|TUN-INSIDE- CP|`169.54.0.2`|
+|CP Security Gateway ASN|`65002`|
+
+## Configuring Cloud VPN
+
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: *If* you want to tie in all the config below, add some intro text here e.g. (general preview of what's next): 

To configure Cloud VPN, you need to configure Cloud Router to exchange routes between your VPC and on-premise networks and then configure a VPN gateway and tunnel for the gateway-Check Point connection.

**Comment by ashishverm** [comment]
Done. PTAL

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 31]
```diff
@@ -0,0 +1,387 @@
...
+1. Open SmartConsole > **Global Properties** > **VPN** > **Advanced**.
+2. Select **Enable VPN Directional Match in VPN Column**.
+
+![alt_text](Image_20.PNG)
+
+**Note:** This is not relevant for a Policy Based scenario.
+
+3. Add these directional match rules in the VPN column for every firewall rule related to VPN traffic:
+
+    Internal_clear > Google Cloud VPN community name (VPN_Community)
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Format. 

`Internal_clear > Google Cloud VPN community name (VPN_Community) Google Cloud VPN community name > Google Cloud VPN community name (VPN_Community) Google Cloud VPN community name (VPN_Community) > Internal_clear`

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 32]
```diff
@@ -0,0 +1,387 @@
...
+    set bgp external remote-as 65000 on
+    set bgp external remote-as 65000 peer 169.254.0.1 on
+    set bgp external remote-as 65000 peer 169.254.0.1 as-override on
+    set bgp external remote-as 65000 peer 169.254.0.1 holdtime 30
+    set bgp external remote-as 65000 peer 169.254.0.1 keepalive 10
+    set inbound-route- filter bgp-policy 512 based-on- as as 65000 on
+    set inbound-route- filter bgp-policy 512 accept-all- ipv4
+    set route-redistribution to bgp-as 65000 from interface eth1 on
+
+**Step 10**. Configure Directional Rules for Route-Based Scenario
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Configure Directional Rules for Route-Based Scenario.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 33]
```diff
@@ -0,0 +1,387 @@
...
+
+This example refers to IKEv2 specifically. You can also use IKEv1 in this scenario.
+
+![alt_text](Image_18.PNG)
+
+**Step 8**. Go to the **Advanced tab**. You can modify the more advanced settings for Phase 1 Phase 2 there.
+
+![alt_text](Image_19.PNG)
+
+**Step 9**. Setup for BGP Deployment
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Setup for BGP Deployment.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 34]
```diff
@@ -0,0 +1,387 @@
...
+
+With route based VPN both static and dynamic routing can be used. This example will use
+dynamic routing. [Cloud Router](https://cloud.google.com/router/docs/) is used to establish
+BGP sessions between the 2 peers.
+
+### Configuring Cloud Router
+
+Step 1: In Google Cloud Platform Console, select **Networking** > **[Cloud Routers](https://console.cloud.google.com/interconnect/routers)** > **Create Router**.
+
+Enter the parameters as shown in the following table and click **Create**.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Step 2.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 35]
```diff
@@ -0,0 +1,387 @@
...
+|Description|           |Description of the cloud router.|
+|Network|`to-cp`|The GCP network the cloud router attaches to. Note: This is the network which manages route information.|
+|Region|`europe-west1`|The home region of the cloud router.Note: Make sure the cloud router is in the same region as the sub-networks it is connecting to.|
+|Google ASN|`65000`|The Autonomous System Number assigned to the cloud router. Use any unused private ASN (64512 - 65534, 4200000000 – 4294967294).|
+
+![alt_text](Image_8.PNG)
+
+### Configuring Cloud VPN
+
+Step 2: In Google Cloud Platform Console, select **Networking** > **Interconnect** > **[VPN](https://console.cloud.google.com/interconnect/vpn)** >> **CREATE VPN CONNECTION**.
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Step 1. 

Remove extra  > here: ** >> **CREATE VPN

File: tutorials/using-cloud-vpn-with-checkpoint/index.md [hunk 36]
```diff
@@ -0,0 +1,387 @@
...
+
+Enter the parameters as shown in the following table for the Google Compute Engine VPN gateway:
```
Comment (deb-saunders) [inline tutorials/using-cloud-vpn-with-checkpoint/index.md]: Step 2.

File: tutorials/using-cloud-vpn-with-checkpoint/index.md (hunk 32 above)
Comment (ashishverm) [reply tutorials/using-cloud-vpn-with-checkpoint/index.md]: Done. Thanks for the detailed feedback.
Really appreciate it.

**Comment by ashishverm** [comment]
Done. Thanks for the detailed feedback.
Really appreciate it.

**Comment by jmdobry** [comment]
@ashishverm Is there anything left that you needed to on this? Or can I merge?

**Comment by ashishverm** [comment]
we can merge now. No pending changes from my end.

**Comment by jmdobry** [comment]
Published at https://cloud.google.com/community/tutorials/using-cloud-vpn-with-checkpoint PTAL

**Comment by ashishverm** [comment]
for some reason Prerequisites section is not rendering properly. Can you please check ?
I created https://github.com/GoogleCloudPlatform/community/pull/350 to correct spacing in case that helps.

**Comment by NumenDivinum21** [comment]
@ashishverm is it possible to configure CheckPoint-to-GoogleCloud Route-based VPN with static routing? Could you please provide an example for such a configuration?
I can't make it work - where to configure TUN-INSIDE GCP address for static Route-based VPN?

**Comment by globeone** [comment]
There is are a number of steps missing from this guide in the Policy Based Site to Site VPN on the Checkpoint side. With these steps the encryption domains will match and the connection will start.

Steps to create an Interoperable Device with the correct settings:

1.  **New... More... Network Object... More... Interoperable Device**
2.  General Properties... Name: **GoogleCloudVPNGateway** (or you can use the actual hostname)
3.  IPv4 Address: **35.195.227.26**
4.  Topology
5.  New...
6.  Name: **toGoogle**
7.  IPv4 Address: 35.195.227.26
8.  Net Mask: 255.255.255.255
9.  **Topology** the little horizontal tab at the top of the current dialog box 
10. **External (leads out to the Internet)**
11. **OK**
12. **New...**
13. Name: **toInternalNetwork**
14. IPv4 Address: **10.0.0.0**
15. Net Mask: **255.255.255.0**
16. **Topology**
17. **Internal (leads to the local network)**
18. IP addresses behind this interface: **Network defined by the interface IP and Netmask**
19. **OK**
20. **OK**

2019-05-03CheckpointR80 10_toGoogle_Topology
2019-05-03CheckpointR80 10_toInternalNetwork_Topology
2019-05-03CheckpointR80 10__interoperable_toInternalNetwork
2019-05-03CheckpointR80 10__interoperable_toGoogle
2019-05-03CheckpointR80 10_edit_interoperable_topology
2019-05-03_CheckpointR80 10_create_interoperable_device

**Comment by ToddKopriva** [comment]
I created a separate bug report in the Issues section to track this, @globeone .

Please continue the conversation there.

**Pull request merged**